perun-core/src/main/java/cz/metacentrum/perun/core/implApi/modules/attributes/AbstractMembershipExpirationRulesModule.java

```diff
@@ -133,6 +135,14 @@ public void checkAttributeSyntax(PerunSessionImpl sess, T entity, Attribute attr
 						parameter + "': " + attrValue.get(parameter));
 			}
 		}
+
+		parameter = doNotExpireSponsoredMembers;
+		if(keys.contains(parameter)) {
+			Matcher dateMatcher = expireSponsoredMembersPattern.matcher(attrValue.get(parameter));
```

Comment:
I don't think "dateMatcher" is the right name for this variable.

Reply:
Fixed.